Simplenote.xcodeproj/project.pbxproj

```diff
@@ -1993,7 +1993,7 @@
 				};
 			};
 			buildConfigurationList = E29ADD3317848E8500E55842 /* Build configuration list for PBXProject "Simplenote" */;
-			compatibilityVersion = "Xcode 3.2";
+			compatibilityVersion = "Xcode 11.0";
```

Comment:
Updating this for same reason as https://github.com/woocommerce/woocommerce-ios/pull/3275#discussion_r539633982, i.e. since CocoaPods 1.10 now use `compatibilityVersion` to decide if `inputFileList`/`outputFileList` is available, we need that bump here to avoid a bigger diff.

Reply:
😮 this explains why I got the full list in [this commit on Simplenote macOS with CP 1.10](https://github.com/Automattic/simplenote-macos/pull/743/commits/5456c8bf2d24858ca0f1a94fb0593bc73aba72d2).

Thank you for documenting this 👍